howto/consume-topic/README.md

```diff
@@ -34,6 +34,50 @@ To deploy this into a Kubernetes cluster, fill in the `metadata` connection deta
 
 ## Subscribe to topics
 
+Dapr allows two methods by which you can subscribe to topics: programatically and declaratively.
```

Comment:
Consider sharper contrast and active voice. For example: 

Dapr supports two methods for creating topic subscription:

* programmatic - where the intend to subscribe to specific topic is defined in user code
* declarative - where the subscription is defined through external configuration

Reply:
And link these to the corresponding sections

Reply:
The sections are right under that sentence..

Reply:
> Consider sharper contrast and active voice. For example:
> 
> Dapr supports two methods for creating topic subscription:
> 
> * programmatic - where the intend to subscribe to specific topic is defined in user code
> * declarative - where the subscription is defined through external configuration

Changed.